pkg/devconsole/proxy/proxy.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/openshift/console/pkg/serverutils"
 )
 
+var insecureClient = &http.Client{
+	Transport: &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
+		},
+	},
+}
+var secureClient = &http.Client{
+	Transport: &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
+	},
+}
```

Comment:
`http.Client`s are not intended to be created per-request. They maintain socket caching and maintain internal state. Since they're concurrency safe, we can simply create two of them at the module level and reference them as necessary per-request